ddtrace/internal/telemetry/writer.py

```diff
@@ -431,45 +432,21 @@ def _app_started_event(self):
         if self._forked:
             # app-started events should only be sent by the main process
             return
+        #  List of configurations to be collected
+        self.add_configurations(
+            [
+                ("data_streams_enabled", config._data_streams_enabled, "unknown"),
+                ("appsec_enabled", config._appsec_enabled, "unknown"),
+                ("propagation_style_inject", str(config._propagation_style_inject), "unknown"),
+                ("propagation_style_extract", str(config._propagation_style_extract), "unknown"),
```

Comment:
These configurations are currently ignored by telemetry intake. Can we rename them to match these values: https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json#L245-L246.